### PR TITLE
PRIME-1048 Manual Adjudication for L2 users

### PR DESCRIPTION
**Pull request by mtpultz**

### CODE REVIEW

**Review by AnaisHebert — APPROVED** [review]
🍔

**Comment by mtpultz** [comment]
LGTM!!! 🍭

File: prime-dotnet-webapi/Controllers/EnrolleesController.cs [hunk 1]
```diff
@@ -616,6 +632,35 @@ public async Task<ActionResult<string>> getSelfDeclarationDocument(int enrolleeI
...
         }
 
+        // GET: api/Enrollees/{enrolleeId}/identification-document/{identificationDocumentId}
+        /// <summary>
+        /// Get the Identification Document download token.
+        /// </summary>
+        /// <param name="enrolleeId"></param>
+        /// <param name="identificationDocumentId"></param>
+        [HttpGet("{enrolleeId}/identification-document/{identificationDocumentId}", Name = nameof(GetIdentificationDocument))]
+        [ProducesResponseType(typeof(ApiBadRequestResponse), StatusCodes.Status400BadRequest)]
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Controllers/EnrolleesController.cs:642]: this should probably be admin only

File: prime-dotnet-webapi/Controllers/EnrolleesController.cs [hunk 2]
```diff
@@ -126,12 +126,28 @@ public async Task<ActionResult<Enrollee>> CreateEnrollee(EnrolleeCreatePayload p
                 return BadRequest(ApiResponse.BadRequest(this.ModelState));
             }
 
+            string filename = null;
+            if (payload.IdentificationDocumentGuid != null)
```
Comment (james-hollinger) [inline prime-dotnet-webapi/Controllers/EnrolleesController.cs:130]: if the user has a BC services card, we should ignore the identification document GUID entirely

**Pull request merged**